packages/wonder-stuff-server-google/src/get-default-log-metadata.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+import type {AppEngineInfo} from "./types.js";
+import {getAppEngineInfo} from "./get-app-engine-info.js";
+
+/**
+ * Get the default log metadata for a Google Cloud app engine server.
+ */
+export const getDefaultLogMetadata = (): {
```

Comment:
Copied from render-gateway's `create-logger.js` in `shared` and typed more strongly.